requirements.txt

```diff
@@ -1,3 +1,3 @@
-zeroc-ice==3.6.5
+zeroc-ice36-python==3.6.5
```

Comment:
I don't quite understand this change, can you explain?

Reply:
Sorry, I mixed up conda and pip package names on this one (https://pypi.org/project/zeroc-ice/ vs https://anaconda.org/ome/zeroc-ice36-python. Will add a commit reverting